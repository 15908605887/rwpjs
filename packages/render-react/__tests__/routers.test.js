const path = require('path')
const routers = require('../src/models/routers')

test('test recursive files directory.',  function(done ) {
    routers.getSrcRouter(path.join(__dirname, 'pages'),function(element){
        return Object.assign(element,{
            path: element.path.replace(__dirname, '')
        })
    }).then(function(files){
        expect(files).toMatchSnapshot();
        done()
    })
})