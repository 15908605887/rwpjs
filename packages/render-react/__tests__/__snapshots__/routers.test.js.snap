// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test recursive files directory. 1`] = `
Array [
  Object {
    "childrens": Array [
      Object {
        "path": "\\\\pages\\\\admin\\\\index",
        "type": "file",
      },
    ],
    "path": "\\\\pages\\\\admin",
    "type": "directory",
  },
  Object {
    "path": "\\\\pages\\\\file1",
    "type": "file",
  },
]
`;
